src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write code here
+const bodyOfTable = document.querySelector('tbody');
+
+const headOfTable = document.querySelector('tHead > tr');
+
+const tableRows = bodyOfTable.rows;
+
+headOfTable.addEventListener('click', () => {
+  let arrayByName;
+
+  switch (true) {
+    case event.target.cellIndex === 0:
+      arrayByName = [...tableRows].sort((a, b) => {
+        return a.children[0].innerText.localeCompare(b.children[0].innerText);
+      });
+      break;
+
+    case event.target.cellIndex === 1:
+      arrayByName = [...tableRows].sort((a, b) => {
+        return a.children[1].innerText.localeCompare(b.children[1].innerText);
+      });
+      break;
```

Comment:
you are using `switch case` as `if`, it's not correct. you can combine similar logic in one `case` and use `event.target.cellIndex` as index
```
const index = event.target.cellIndex;
switch (index) {
  case 0:
  case 1:
    a.children[index].localeCompare(b.children[index];
     ...
}
```

Reply:
> you are using `switch case` as `if`, it's not correct. you can combine similar logic in one `case` and use `event.target.cellIndex` as index
> 
> ```
> const index = event.target.cellIndex;
> switch (index) {
>   case 0:
>   case 1:
>     a.children[index].localeCompare(b.children[index];
>      ...
> }
> ```

Done